config.js

```diff
@@ -72,10 +72,10 @@ const DEFAULT_JOB_QUEUE_TTL_SECONDS_ASYNC = 3600;
 const DEFAULT_JOB_QUEUE_TTL_SECONDS_SYNC = 25;
 
 // default set to 30 minutes
-const DEFAULT_JOB_REMOVAL_DELAY_SECONDS = 1800;
+const DEFAULT_JOB_REMOVAL_INTERVAL_SECONDS = 1800;
 
-const JOB_REMOVAL_DELAY_SECONDS = pe.KUE_JOBS_REMOVAL_DELAY ||
-  DEFAULT_JOB_REMOVAL_DELAY_SECONDS;
+const JOB_REMOVAL_INTERVAL_SECONDS = pe.KUE_JOBS_REMOVAL_INTERVAL ||
```

Comment:
since you're changing the name of the env var, you need to make sure that prod and sandbox are configured appropriately... you can add the new one anytime (before the release) but also need to make sure to remove the old one after the release